elements/rh-surface/docs/00-overview.md

```diff
@@ -0,0 +1,45 @@
+## Overview
+
+{{ tagName | getElementDescription }}
+
+<rh-surface class="sample" style="--columns:3;" color-palette="darkest">
+{# {% sample columns=3, code="show", colorPalette="darkest" %} #}
+  <rh-blockquote>
+    <p>Surface is used to provide a theme to children</p>
+  </rh-blockquote>
+  <rh-card color-palette="dark">
```

Comment:
Not sure if this needs its own color-palette since the surface should supply it, unless we also want to show how a nested card can have its own color palette?